data-pipeline/src/pipeline/pre_processing.py

```diff
@@ -792,7 +776,6 @@ def build_academy_data(
             "LA (code)": "LA Code",
             "LA (name)": "LA Name",
             "Academy Trust Name": "Trust Name",
```

Comment:
Academy Trust Name and Academy Trust UPIN will need updating based on AAR and AAR_CS

Company_Name & Lead_UPIN

Reply:
Going to run through the data at this point and update accordingly: `URN_x` isn't there, for instance; `Trust UPIN` will appear following the merge onto `central_services` in a dozen lines or so.

Reply:
- `URN_x` and `UKPRN_x` removed (no longer present)
- `Company_Name` -> `Trust Name` moved to `prepare_aar_data`
- `config.income_category_map["academies"]` and ` config.cost_category_map["academies"]` removed as this already takes place earlier in `prepare_aar_data`